postgres/src/model.py

```diff
@@ -0,0 +1,12 @@
+from typing import List
+
+from pydantic import BaseModel
+
+
+class Task(BaseModel):
+    id: int
+    task: str
+
+
+class Tasks(BaseModel):
```

Comment:
You don't necessarily neeed this. If you want to return list of tasks. You can define `response_model=List[Task]` and fastapi will do it's thing.